packages/keychain/src/utils/connection/connect.ts

```diff
@@ -11,7 +11,7 @@ export function connectFactory({
   setContext: (context: ConnectionCtx) => void;
 }) {
   return (origin: string) =>
-    (rpcUrl: string): Promise<ConnectReply> => {
+    (_: Policy[], rpcUrl: string): Promise<ConnectReply> => {
```

Comment:
I think we want to add back the `setPolicy` stuff here i order to make it backwards compatible right?

Reply:
Ah I had assumed u were pulling those from url, yeah let me add them back

Reply:
We are and I think contorller 0.3.46 actually adds them to the url already too right?

Maybe we don't need it then